### DISCUSSION TITLE
veSTG on BSC

### DISCUSSION THREAD

**Discussion by jpbuss** [snapshot-labs/snapshot#3252] [Q&A] [answered]
hi
I have veSTG on the BSC, I can't vote?

**Comment by metrox-eth** [comment, marked as answer]
You need to hold the tokens before the creation of the proposal (snapshot). You will be able to vote on future proposals.

**Reply by jpbuss** [reply]
merci

**Reply by metrox-eth** [reply]
Y a pas de quoi !

**Reply by NPSF1** [reply]
thankss

**Comment by Philipp797** [comment]
> hi I have veSTG on the BSC, I can't vote?

the same problem
may be we need to wait? could anybody help us?

**Reply by metrox-eth** [reply]
This question has already been answered, plz look above
